src/main/java/eggventory/StockList.java

```diff
@@ -32,6 +32,33 @@ public StockList() {
         return stockList;
     }
 
+    /**
+     *
```

Comment:
where is the description ahaha

Reply:
``OUTDATED``

Reply:
well i see travis also asked you the same question